src/components/cursor.js

```diff
@@ -250,6 +261,24 @@ module.exports.Component = registerComponent('cursor', {
       evt.preventDefault();
     }
 
+    if (this.data.rayOrigin === 'xrselect' && evt.type === 'selectstart') {
+      this.onMouseMove(evt);
+      this.el.components.raycaster.checkIntersections();
+
+      // if something was tapped on don't do ar-hit-test things
+      if (
+        this.el.components.raycaster.intersectedEls.length &&
+        this.el.sceneEl.components['ar-hit-test'] !== undefined &&
+        this.el.sceneEl.components['ar-hit-test'].data.enabled
+      ) {
+        // Cancel the ar-hit-test behaviours and disable the ar-hit-test
+        this.el.sceneEl.components['ar-hit-test'].data.enabled = false;
```

Comment:
We shouldn't access data directly but using getAttribute:

`this.el.sceneEl.getAttribute('ar-hit-test).enabled`